.github/workflows/beta.yaml

```diff
@@ -88,11 +107,121 @@ jobs:
       - name: Push Docker image
         uses: docker/build-push-action@v4
         with:
+          builder: ${{ steps.buildx.outputs.name }}
           build-args: |
-            CCACHE_REMOTE_STORAGE=${{ secrets.PORLA_CCACHE_REMOTE_STORAGE_WRITE }}
```

Comment:
Is there any reason to remove this? Keeping ccache would greatly improve build times